tests/ert3/config/test_stages_config.py

```diff
@@ -148,140 +128,74 @@ def test_step_unknown_script(tmpdir):
         ert3.config.load_stages_config(config)
 
 
-def test_step_function_definition_error():
+def test_step_function_definition_error(base_function_stage_config):
+    config = base_function_stage_config
+    config[0]["function"] = "builtinssum"
     with pytest.raises(
         pydantic.error_wrappers.ValidationError,
         match=r"Function should be defined as module:function",
     ):
-        ert3.config.load_stages_config(
-            [
-                {
-                    "name": "minimal_function_stage",
-                    "type": "function",
-                    "output": [{"record": "some_record", "location": "some_location"}],
-                    "function": "builtinssum",
-                }
-            ]
-        )
-
-
-def test_step_function_error():
+        ert3.config.load_stages_config(config)
+
+
+def test_step_function_error(base_function_stage_config):
+    config = base_function_stage_config
+    config[0]["function"] = "builtins:sun"
     with pytest.raises(
```

Comment:
Is this formatted over three lines on purpose?

Reply:
not sure it might be black that did this...

Reply:
definitely, maybe some argument was after it before, and the comma was not removed.